README.md

```diff
@@ -95,6 +95,36 @@ KubeKey can install Kubernetes and KubeSphere together. The dependency that need
 > * Docker needs to be installed before building.
 > * If you have problem to access `https://proxy.golang.org/`, excute `build.sh -p` instead.
 
+### YUM
```

Comment:
Is it available now?  Would it be better to add this section to `README.md` after the next release?

Reply:
Yes. I agree with you. It might make people feel confused if it's unavailable now. Another solution is that we publish [v1.0.1](https://github.com/kubesphere/kubekey/releases/tag/v1.0.1) before this PR gets merged.

Reply:
Forget it. It will mass up the git history if I do that. I already made it become invisible via commenting on it. We can open it after the next release of kk.

Reply:
ok